python.Rproj

```diff
@@ -0,0 +1,18 @@
+Version: 1.0
```

Comment:
Should this workspace file be committed or listed in `.gitignore`?

Reply:
Since we have the [Markdown Notebook infrastructure](https://github.com/datatrail-jhu/python/pull/13) set up, we can delete this file. I mainly created this file to [work in RStudio](https://support.posit.co/hc/en-us/articles/200526207-Using-RStudio-Projects).